src/chartkit/components/VerticalBar.js

```diff
@@ -42,12 +43,16 @@ class VerticalBar extends Component {
 
     if (data) {
       const { index, indexValue } = data;
-      this.setState({ highlightedIndex: index, highlightedIndexValue: indexValue });
+      const label = {
```

Comment:
I would just use `data.data` here that way we grab something like `{id: "agg5to10", label: "1 - 5", value: 450}` and can transform that however we want downstream

Reply:
Changed to:
```javascript
const { id, index, indexValue, value } = data;
this.setState({ highlightedIndex: index, highlightedIndexValue: indexValue });
if (this.props.analyticsTracking) {
  trackUserInteraction({
    category: `${this.props.analyticsTracking.category}: ${this.props.analyticsTracking.subcategory}`,
    action: `Chart Bar: ${TRACKING_EVENTS.actions.hover}`,
    label: JSON.stringify({ id, indexValue, value }),
  });
}
```